integration-tests/r_integration_test/rclipper_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+redis_port=$1
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd $DIR
+
+version_tag=$(<VERSION.txt)
+
+CLIPPER_ROOT=$DIR/../..
+cd $CLIPPER_ROOT
+
+time docker build -t clipper/r-container-base:$version_tag -f ./dockerfiles/RContainerDockerfile ./
```

Comment:
Delete lines 13-18 (`version_tag...` through `time docker build...`). We don't want duplicate calls to build docker images scattered around the code.

You can assume that `./bin/build_docker_images.sh` has been called before this test is run. The Python integration tests all assume that.